tests/unit/test_events.py

```diff
@@ -174,6 +188,66 @@ def test_parse_release(self, github_release_webhook):
             event_object.project_url == "https://github.com/packit-service/hello-world"
         )
 
+    def test_parse_mr(self, merge_request):
+        event_object = Parser.parse_event(merge_request)
+
+        assert isinstance(event_object, MergeRequestGitlabEvent)
+        assert event_object.trigger == TheJobTriggerType.pull_request
+        assert event_object.action == MergeRequestAction.opened
+        assert event_object.object_id == 58759529
+        assert event_object.object_iid == 1
+        assert event_object.source_repo_namespace == "testing-packit"
+        assert event_object.source_repo_name == "hello-there"
+        assert event_object.commit_id == "1f6a716aa7a618a9ffe56970d77177d99d100022"
+        assert event_object.target_repo_namespace == "testing-packit"
+        assert event_object.target_repo_name == "hello-there"
+        assert event_object.https_url == "https://gitlab.com/testing-packit/hello-there"
+
+        assert isinstance(event_object.project, GithubProject)
+        assert event_object.project.full_repo_name == "packit-service/packit"
+        assert (
+            not event_object.base_project  # With Github app, we cannot work with fork repo
+        )
```

Comment:
This is probably not relevant to gitlab, but we need to check if we need to check this:
- If we don't fill this object, we need to do everything (comments, commit-status, getting config-file content from the PR commit) from upstream.
- Or have in mind (and check) if we have rights to do all of the OGR usage without having rights to the fork repo.

Reply:
Gotcha!